src/hooks/useGetColonyAction.ts

```diff
@@ -43,16 +43,24 @@ const useGetColonyAction = (colony?: Colony | null) => {
     setIsPolling(false);
   }
 
-  const { data: motionStateData, loading: loadingMotionState } =
-    useGetMotionStateQuery({
-      skip: !action?.motionData,
-      variables: {
-        input: {
-          colonyAddress: colony?.colonyAddress ?? '',
-          motionId: Number(action?.motionData?.motionId),
-        },
+  const {
+    data: motionStateData,
+    loading: loadingMotionState,
+    refetch: refetchMotionState,
+  } = useGetMotionStateQuery({
+    skip: !action?.motionData,
+    variables: {
+      input: {
+        colonyAddress: colony?.colonyAddress ?? '',
+        motionId: Number(action?.motionData?.motionId),
       },
-    });
+    },
+  });
+
+  /* Ensures motion state is kept in sync with motion data */
+  useEffect(() => {
```

Comment:
This ensures a smooth transition to the voting phase after both sides have been staked 100%